Client/Ecosia/UI/MyImpact/MultiplyImpact.swift

```diff
@@ -0,0 +1,228 @@
+        
+        firstStep.topAnchor.constraint(equalTo: flowTitle.bottomAnchor, constant: 12).isActive = true
+        firstStep.leftAnchor.constraint(equalTo: content.leftAnchor).isActive = true
+        firstStep.rightAnchor.constraint(equalTo: content.rightAnchor).isActive = true
+        
+        secondStep.topAnchor.constraint(equalTo: firstStep.bottomAnchor, constant: 20).isActive = true
+        secondStep.leftAnchor.constraint(equalTo: content.leftAnchor).isActive = true
+        secondStep.rightAnchor.constraint(equalTo: content.rightAnchor).isActive = true
+        
+        thirdStep.topAnchor.constraint(equalTo: secondStep.bottomAnchor, constant: 20).isActive = true
+        thirdStep.leftAnchor.constraint(equalTo: content.leftAnchor).isActive = true
+        thirdStep.rightAnchor.constraint(equalTo: content.rightAnchor).isActive = true
+        
+        inviteFriends.leftAnchor.constraint(equalTo: content.leftAnchor, constant: 16).isActive = true
+        inviteFriends.rightAnchor.constraint(equalTo: content.rightAnchor, constant: -16).isActive = true
+        inviteFriends.heightAnchor.constraint(equalToConstant: 44).isActive = true
+        inviteFriends.topAnchor.constraint(greaterThanOrEqualTo: thirdStep.bottomAnchor, constant: 16).isActive = true
+        
+        let contentHeight = content.heightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.heightAnchor)
+        contentHeight.priority = .defaultLow
+        contentHeight.isActive = true
+        
+        let cardHeight = card.heightAnchor.constraint(equalToConstant: 0)
+        cardHeight.priority = .defaultHigh
+        cardHeight.isActive = true
+        
+        applyTheme()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        Analytics.shared.openInvitations()
+    }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        applyTheme()
+    }
+    
+    func applyTheme() {
+        view.backgroundColor = .theme.ecosia.modalBackground
+        subtitle?.textColor = .theme.ecosia.secondaryText
+        card?.backgroundColor = .theme.ecosia.impactMultiplyCardBackground
+        card?.layer.borderColor = UIColor.theme.ecosia.impactMultiplyCardBorder.cgColor
+        cardIcon?.image = UIImage(themed: "impactReferrals")
+        cardTitle?.textColor = .theme.ecosia.highContrastText
+        cardSubtitle?.textColor = .theme.ecosia.secondaryText
+        flowTitle?.textColor = .theme.ecosia.secondaryText
+        
+        dash?.applyTheme()
+        firstStep?.applyTheme()
+        secondStep?.applyTheme()
+        thirdStep?.applyTheme()
+    }
+    
+    @objc private func learnMore() {
+        delegate?.ecosiaHome(didSelectURL: URL(string: "https://ecosia.zendesk.com/hc/en-us/articles/4406431901714-How-does-inviting-friends-to-Ecosia-work-")!)
```

Comment:
Would be nice to have this in core like the other urls.. also shouldn't it be localized? :)

Reply:
Yes, I think that would be the ideal. At the moment we only have the english version of it. I would prefer to keep this here until we can localise it and add it with the rest of the urls.

Reply:
ok - fine for me :) It'll probably pop up in QA for the case we forget ;)